pages/science/reactions.rst

```diff
@@ -285,6 +285,57 @@ Chebyshev reactions can be defined in the CTI format using the
 :cti:class:`chebyshev_reaction` entry, or in the YAML format using the
 :ref:`Chebyshev <sec-yaml-Chebyshev>` reaction ``type``.
 
+.. _sec-Blowers-Masel:
+
+Blowers-Masel Reactions
+-----------------------
+
+**New in Cantera 2.6**
+
+In some circumstances like thermodynamic sensitivity analysis, or
+modeling heterogeneous reactions from one catalyst surface to another,
+the enthalpy of reactions needs to be tweaked. Since reactions' energetics
+is changed, the activation energy of reactions should also be adjusted accordingly
+to provide accurate simulation results. The enthalpy can be modified by changing
+temperature and revising NASA polynomial coefficients in Cantera. To predict the
+activation energy change, the Blowers-Masel rate expression is implemented, which
+is an approximation proposed by Blowers and Masel [#BlowersMasel2000]_ to automatically
+scale activation energy as species' enthalpies are changed in hydrogen transfer reactions.
```

Comment:
```suggestion
the enthalpy change of a reaction (:math:`\Delta H`) can be modified. Due to the change in :math:`\Delta H`,
the activation energy of the reaction must be adjusted accordingly to provide accurate simulation results. To
adjust the activation energy due to changes in the reaction enthalpy, the Blowers-Masel rate expression is
available. This approximation was proposed by Blowers and Masel [#BlowersMasel2000]_ to automatically
scale activation energy as the reaction enthalpy is changed.
```

A couple of notes about this suggestion:
1) It is not clear from the text why the last sentence switches to species enthalpies, when the rest of the paragraph is about reaction enthalpy.
2) Does this only apply to hydrogen transfer reactions? If so, there's no code in the implementation to check and limit that, as far as I know.

Reply:
1. Sorry, I mean that the enthalpy of species involved in a reaction change leads to a reaction enthalpy change, I will change it to reaction enthalpy for the consistency.
2. Yes, the paper only justified their approximation using hydrogen transfer reactions, but this approximation could be used for other type of reactions as a first guess if the reaction's enthalpy is changed and we have no idea about the activation energy after the enthalpy is changed.